pom.xml

```diff
@@ -246,6 +246,7 @@
                     <configuration>
                         <compilerId>groovy-eclipse-compiler</compilerId>
                         <release>17</release>
+                        <fork>true</fork>
```

Comment:
What is this?

Reply:
Oh. For Lombok. Got it

Reply:
Originally yes, but given all the troubles I had when trying to integrate Lombok, I've removed the dependency so we can revisit at a later time. This can be removed if needed.